.env

```diff
@@ -0,0 +1,2 @@
+DATABASE_USERNAME=postgres
+DATABASE_PASSWORD=myrt13th3turt13
```

Comment:
err was this file supposed to be committed?

Reply:
Yes, supposed to be committed. I suppose I could add a .example version instead and then gitignore it, but since it's part of docker the "password" really doesn't matter. But that's a better change, so it doesn't look like sensitive info is in the repo. I'll make the change, thanks.